eogrow/pipelines/prediction.py

```diff
@@ -137,6 +140,7 @@ def _get_prediction_node(self, previous_node: EONode) -> EONode:
             input_features=self.config.input_features,
             mask_feature=_optional_typed_feature(FeatureType.MASK_TIMELESS, self.config.prediction_mask_feature_name),
             output_feature=(FeatureType.DATA_TIMELESS, self.config.output_feature_name),
+            output_dtype=self.config.dtype,
```

Comment:
Perhaps the `output_` prefix is not necessary here

Reply:
probably not, but i'd still keep it so that it stays more verbose (there are many parameters for the task)